crates/flock/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "flock"
+description = "Locked files that can be used like normal File"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/flock"
+version = "1.0.0"
```

Same here:

```suggestion
version = "0.1.0"
```

The name `flock` is available.